subsystem-drive/source/main.cpp

```diff
@@ -15,14 +19,18 @@ drive_commands SerialEnterCommands()
 int main()
 {
     drive_commands commands;
+    tri_wheel_router_arguments arguments;
     SteerModes steermode;
+    TriWheelRouter tri_wheel;
+    ModeSwitch mode_switch;
+    CommandLerper lerp;
     while (1)
     {
         commands = SerialEnterCommands();
         commands.Print();
         sjsu::Delay(2s);
-
-        tri_wheel_router_arguments arguments = steermode.DriveSteering(commands);
+        
+        arguments = tri_wheel.SetLegArguments(ModeSelect::SelectMode(lerp.Lerp(mode_switch.SwitchSteerMode(commands, arguments))));
```

Comment:
Should make lerp class static like we did for steer modes

Reply:
We can't, the lerp class needs to hold previous information which i'm pretty sure ruins the point of the static.